python/py-hid/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-hid
+version             1.0.5
+
+categories-append   comms
+
+license             MIT
+supported_archs     noarch
+
+maintainers         {@bashu gmail.com:bashu.was.here} openmaintainer
+
+description         Python hidapi bindings in ctypes (aka pyhidapi)
+long_description    {*}${description}
+
+homepage            https://github.com/apmorton/pyhidapi
+
+checksums           rmd160  8afa962c1e67e7d0f06e1cf9d28f8737e8f1e19e \
+                    sha256  1e954e7f7ab9b7c9dfc78db59504692c17db3b71249492b976b1525b97dbb0e8 \
+                    size    3881
+
+patchfiles          hid-__init__.py.patch
+
+python.versions     34 35 36 37 38 39 310
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+        port:py${python.version}-setuptools
+
+    depends_lib     port:hidapi
+
+    livecheck.type  none
```

Comment:
You can remove this line as it is already set by the python PG

Reply:
fixed